pkgs/development/libraries/libmongocrypt/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, lib, fetchFromGitHub, fetchpatch, cmake, pkg-config, mongoc, openssl }:
+stdenv.mkDerivation rec {
+  pname = "libmongocrypt";
+  version = "1.7.4";
+
+  src = fetchFromGitHub {
+    owner = "mongodb";
+    repo = pname;
+    rev = version;
+    hash = "sha256-I4KG2BHAovin9EaF8lNzJzucARvi0Qptz5Y9gTt3WkE=";
+  };
+
+  patches = [
+    (fetchpatch {
```

Comment:
```suggestion
    # fix cmake file
    (fetchpatch {
```

Reply:
I'm kinda hoping it just gets upstreamed and we can drop this entirely, but we'll see.